### PR TITLE
Add jivosite widget: make widget optional, move widgetID to settings

### PR DESCRIPTION
**Pull request by wowkalucky**
Made assumption: settings object is presented in django template "lms/templates/main_django.html"

### CODE REVIEW

**Comment by dgamanenko** [comment]
@wowkalucky 
please squash commits

File: lms/envs/common.py [hunk 1]
```diff
@@ -2967,4 +2967,7 @@
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
 #####RACCOONGANG########
 DOC_LINK_BASE_URL = ''
+
+# JIVOSITE widget settings:
+JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID')
```
Comment (z4y4ts) [inline lms/envs/common.py]: I thought you wanted to get it from config file rather than from environment

Comment (wowkalucky) [inline lms/envs/common.py]: But, from devops talk I was sure there is one true way to get some settings-param: yaml > json > env > settings > code. So, we should discuss it...

Comment (dgamanenko) [inline lms/envs/common.py]: @wowkalucky 
it's correct, just add a default setting for JIVOSITE_WIDGET_ID variable.
 - for example: JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID', '')
and prepare pull request into configuration repository
 - as an example: https://github.com/raccoongang/configuration/commit/51a12cd84874fd1d8d88461205a9f113466a729e

Comment (dgamanenko) [inline lms/envs/common.py]: @wowkalucky also you can read from json without os.environ: as an example: https://github.com/raccoongang/edx-platform/blob/6f4fc73d639751c0d596374a051ced81bf90f267/lms/envs/aws.py#L256

**Comment by wowkalucky** [comment]
@dgamanenko, kill the (https://github.com/raccoongang/configuration/pull/27), please, and make sure new settings are available:
- FEATURES.get('ENABLE_JIVOSITE')
- FEATURES.get('JIVOSITE_WIDGET_ID')

**Pull request merged**